src/components/ItemTable/ItemAvailability.test.tsx

```diff
@@ -13,10 +15,153 @@ import {
   itemUnavailable,
 } from "../../../__test__/fixtures/itemFixtures"
 import { searchResultPhysicalItems } from "../../../__test__/fixtures/searchResultPhysicalItems"
+import { notDeepEqual } from "assert"
 
 const parentBib = new SearchResultsBib(searchResultPhysicalItems)
 
 describe("ItemAvailability", () => {
+  describe("special collections", () => {
+    it("onsite aeon finding aid", () => {
+      const item = new Item(itemPhysicallyRequestable, parentBib)
+      item.availability = new ItemAvailabilityModel({
```

Comment:
This gets called automatically in `Item` when it's initialized. Are you doing it here just for the tests?

Reply:
yea I put in in here so it would be explicit per test exactly what case was being tested. i find it messy and hard to maintain specific actual fixtures for all these different cases.

Reply:
I suppose I could have just updated `item.availabilty.key` explicitly for each one.